docs/architecture/raft.rst

```diff
@@ -175,13 +313,68 @@ Examples
    * - 1-4 nodes
      - Schema updates are possible and safe.
      - Try restarting the nodes. If the nodes are dead, :doc:`replace them with new nodes </operating-scylla/procedures/cluster-management/replace-dead-node-or-more/>`.
-   * - 1 DC 
+   * - 1 DC
      - Schema updates are possible and safe.
      - When the DC comes back online, try restarting the nodes in the cluster. If the nodes are dead, :doc:`add 3 new nodes in a new region </operating-scylla/procedures/cluster-management/add-dc-to-existing-dc/>`.
    * - 2 DCs
      - Cluster is not fully operational. The data is available for reads and writes, but schema changes are impossible.
-     - When the DCs come back online, restart the nodes. If at least one DC fails to come back online and the nodes are lost, :doc:`restore the latest cluster backup into a new cluster </operating-scylla/procedures/backup-restore/restore/>`. You can contact `ScyllaDB support <https://www.scylladb.com/product/support/>`_ for assistance.
-     
+     - When the DCs come back online, restart the nodes. If at least one DC fails to come back online and the nodes are lost, consult the :ref:`manual Raft recovery section <recover-raft-procedure>`.
+
+.. _recover-raft-procedure:
+
+Raft manual recovery procedure
+==============================
+
+.. versionadded:: 5.2
+
+The manual Raft recovery procedure applies to the following situations:
+
+* :ref:`The internal Raft upgrade procedure <verify-raft-procedure>` got stuck because one of your nodes failed in the middle of the procedure and is irrecoverable,
+* or the cluster was running Raft but a majority of nodes (e.g. 2 our of 3) failed and are irrecoverable. Raft cannot progress unless a majority of nodes is available.
+
+.. warning::
+
+   Perform the manual recovery procedure **only** if you're dealing with **irrecoverable** nodes. If it is possible to restart your nodes, do that instead of manual recovery.
```

Comment:
Perhaps it also makes sense to clarify here that there is no point in recoverying raft alone if you can't recover some data ranges - and in this cases it makes sense to segway to restoring from a backup.

Reply:
Well, if you have a backup, it means you can recover the data ranges (by uploading the backed-up sstables).

If the user lost a majority of nodes, but they decided to keep the cluster nevertheless and now they are reading this section, should we try to stop them right now (from keeping the cluster)?

Reply:
Another problem when trying to mention restoring a cluster from backup is the fact that we have no documented procedure for doing that :)